src/fields/hooks/afterRead/relationshipPopulationPromise.ts

```diff
@@ -40,16 +40,18 @@ const populate = async ({
     }
 
     if (shouldPopulate) {
-      relationshipValue = await req.payloadDataLoader.load(JSON.stringify([
-        relatedCollection.config.slug,
-        id,
-        depth,
-        currentDepth + 1,
-        req.locale,
-        req.fallbackLocale,
-        overrideAccess,
-        showHiddenFields,
-      ]));
+      relationshipValue = await req.payloadDataLoader.load(
+        {
+          collection: relatedCollection.config.slug,
+          id: id as string,
```

Comment:
type is unknown

Can this be fixed?

Reply:
This one is gonna be trickier to fix. I think we should avoid this one for now.